Content.Server/Chat/Managers/ChatManager.cs

```diff
@@ -201,8 +201,24 @@ public void SendHookOOC(string sender, string message)
 
     public void SendHookAdmin(string sender, string message)
     {
+        // fuck this class
+        var clients = _adminManager.ActiveAdmins.Select(p => p.Channel);
```

Comment:
https://www.youtube.com/watch?v=IAlqG-49hFE

```suggestion
        var clients = _adminManager.ActiveAdmins.Select(p => p.Channel);
```

Reply:
1984